File: isPanagram.java

public class isPanagram {

	public static boolean is_panagram(String input) {
		int[] letter_frequency = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
		char[] input_str = input.toLowerCase().toCharArray();
		for(int i = 0; i <input_str.length; i++) {
	        if(input_str[i] != ' '){
	            letter_frequency[input_str[i] % 97] += 1;
	        }
		}
		for(int i = 0; i <26; i++) {
			if(letter_frequency[i] == 0){
				return false;
	        }
		}
		return true;
	}
	

	public static void main(String[] args) {
		String input = "Pack my box with five dozen liquor jugs";
		System.out.println(is_panagram(input));
	}

}

File: is_panagram.py
def is_panagram(letters_frequency):
    return len(letters_frequency.keys()) == 26


def count_letter_frequency(input):
    letters_frequency = {}
    for i in input:
        if i.isalpha() and i not in letters_frequency:
            letters_frequency[i.lower()] = 1
        elif i.isalpha() and i in letters_frequency:
            letters_frequency[i.lower()] += 1
    return letters_frequency


def check_panagram(input):
    letters_frequency = count_letter_frequency(input)
    return is_panagram(letters_frequency)


input_string = "Pack my box with five dozen liquor jugs"
print(check_panagram(input_string))